applications/StructuralMechanicsApplication/custom_elements/base_shell_element.cpp

```diff
@@ -308,48 +309,13 @@ void BaseShellElement::CalculateDampingMatrix(
     ProcessInfo& rCurrentProcessInfo
     )
 {
-    KRATOS_TRY;
-
-    const SizeType num_dofs = GetNumberOfDofs();
-
-    if ( rDampingMatrix.size1() != num_dofs )
-        rDampingMatrix.resize( num_dofs, num_dofs, false );
-
-    noalias( rDampingMatrix ) = ZeroMatrix( num_dofs, num_dofs );
-
-    // 1.-Calculate StiffnessMatrix:
-
-    MatrixType StiffnessMatrix  = Matrix();
-    VectorType ResidualVector  = Vector();
-
-    CalculateAll(StiffnessMatrix, ResidualVector, rCurrentProcessInfo, true, false);
+    const std::size_t matrix_size = GetNumberOfDofs();
 
-    // 2.-Calculate MassMatrix:
-
-    MatrixType MassMatrix  = Matrix();
-
-    CalculateMassMatrix(MassMatrix, rCurrentProcessInfo);
-
-    // 3.-Get Damping Coeffitients (RAYLEIGH_ALPHA, RAYLEIGH_BETA)
-    double alpha = 0.0;
-    if( GetProperties().Has(RAYLEIGH_ALPHA) )
-        alpha = GetProperties()[RAYLEIGH_ALPHA];
-    else if( rCurrentProcessInfo.Has(RAYLEIGH_ALPHA) )
-        alpha = rCurrentProcessInfo[RAYLEIGH_ALPHA];
-
-    double beta  = 0.0;
-    if( GetProperties().Has(RAYLEIGH_BETA) )
-        beta = GetProperties()[RAYLEIGH_BETA];
-    else if( rCurrentProcessInfo.Has(RAYLEIGH_BETA) )
-        beta = rCurrentProcessInfo[RAYLEIGH_BETA];
-
-    // 4.-Compose the Damping Matrix:
-
-    // Rayleigh Damping Matrix: alpha*M + beta*K
-    noalias( rDampingMatrix ) += alpha * MassMatrix;
-    noalias( rDampingMatrix ) += beta  * StiffnessMatrix;
-
-    KRATOS_CATCH( "" )
+    StructuralMechanicsElementUtilities::CalculateRayleighDampingMatrix(
```

Comment:
I like everything in one line, the text editors can automatically jump line :P

Reply:
hm I am not sure if all editors can do this by default
for VS code I wouldn't know how without searching

plus in the baseclasses the arguments are also multiline usually, e.g. this: https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/includes/condition.h#L243-L245

Or did I misunderstand you?

Reply:
I mean to do the call in one line, OK is only an aesthetic thing